app/javascript/components/visualization/controls/ScatterPlotLegend.js

```diff
@@ -203,6 +295,31 @@ export default function ScatterPlotLegend({
           </>
           }
         </div>
+        { enableColorPicking &&
+          <div>
+            { showColorControls && !isColorSaving &&
+              <>
+                <span>Click a label to select a new color</span><br/>
+                <div>
+                  <a role="button" data-analytics-name="legend-color-picking-save" onClick={saveColors}>
```

Comment:
I think analysts would more likely intuit it a bit differently.

```suggestion
                  <a role="button" data-analytics-name="legend-color-picker-save" onClick={saveColors}>
```